2013/02/22 17:57:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/YiiDemo/
---
2013/02/22 17:58:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/
---
2013/02/22 18:02:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/index.php?r=site/index
---
2013/02/22 18:29:42 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/22 18:30:44 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/22 18:38:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "hello".' in E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hello')
#1 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/YiiDemo/hello
---
2013/02/22 18:40:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "page/about".' in E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('page/about')
#1 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/YiiDemo/page/about
---
2013/02/24 11:18:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "abc".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('abc')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('abc')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/abc')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/abc
---
2013/02/24 11:18:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "abc".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('abc')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('abc')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/abc')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/abc
---
2013/02/24 11:19:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "abc".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('abc')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('abc')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/abc')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/abc
---
2013/02/24 11:21:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "hello".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('hello')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('hello')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/hello')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/site/hello
---
2013/02/24 11:30:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "hello".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('hello')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('hello')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/hello')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/hello
---
2013/02/24 11:31:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "hello".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('hello')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('hello')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/hello')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/hello
---
2013/02/24 11:31:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "hello".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('hello')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('hello')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/hello')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/hello
---
2013/02/24 12:17:20 [error] [php] include(cHtml.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (E:\xampp_new\htdocs\yii\framework\YiiBase.php:421)
Stack trace:
#0 E:\xampp_new\htdocs\YiiDemo\protected\views\website\index.php(10): spl_autoload_call()
#1 E:\xampp_new\htdocs\yii\framework\web\CBaseController.php(126): require()
#2 E:\xampp_new\htdocs\yii\framework\web\CBaseController.php(95): WebsiteController->renderInternal()
#3 E:\xampp_new\htdocs\yii\framework\web\CController.php(869): WebsiteController->renderFile()
#4 E:\xampp_new\htdocs\yii\framework\web\CController.php(782): WebsiteController->renderPartial()
#5 E:\xampp_new\htdocs\YiiDemo\protected\controllers\WebsiteController.php(12): WebsiteController->render()
#6 E:\xampp_new\htdocs\yii\framework\web\actions\CInlineAction.php(49): WebsiteController->actionIndex()
#7 E:\xampp_new\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): WebsiteController->runAction()
#9 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): WebsiteController->runActionWithFilters()
#10 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): WebsiteController->run()
#11 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/website
in E:\xampp_new\htdocs\YiiDemo\protected\views\website\index.php (10)
in E:\xampp_new\htdocs\YiiDemo\protected\controllers\WebsiteController.php (12)
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/24 12:28:49 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/index/n...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/index/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/website/index
---
2013/02/24 12:30:36 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/index/n...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/index/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/website/index
---
2013/02/24 12:30:46 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website
---
2013/02/24 12:30:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "duc".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(270): CController->missingAction('duc')
#1 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('duc')
#2 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/duc')
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/website/duc
---
2013/02/24 12:31:05 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/
---
2013/02/24 12:31:21 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/index/d...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/index/duc
---
2013/02/24 12:44:22 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/page/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/index.php/website/index/name/Duc
---
2013/02/24 12:44:44 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/page/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/index.php/website/index/name/Duc
---
2013/02/24 12:45:04 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/page/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/website
---
2013/02/24 12:46:06 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/page/name/Duc
HTTP_REFERER=http://localhost/YiiDemo/website
---
2013/02/24 12:46:11 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php/website/page/name
---
2013/02/24 12:46:35 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/na...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/name
---
2013/02/24 12:46:40 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/du...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/duc
---
2013/02/24 12:46:51 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/du...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/duc
---
2013/02/24 12:47:15 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/du...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/duc
---
2013/02/24 12:47:28 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/du...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/duc
---
2013/02/24 12:49:22 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page
---
2013/02/24 12:49:42 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page
---
2013/02/24 12:49:45 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/du...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page/duc
---
2013/02/24 14:13:03 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/website/page
---
2013/02/24 14:24:20 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/he...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page/hello
---
2013/02/24 14:24:51 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/24 14:26:32 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/24 14:27:55 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/24 14:28:30 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page/he...')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page/hello
---
2013/02/24 14:28:35 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in E:\xampp_new\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/page')
#5 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/YiiDemo/index.php?r=website/page
---
2013/02/24 17:01:39 [error] [php] strpos() expects parameter 1 to be string, array given (E:\xampp_new\htdocs\yii\framework\web\CController.php:963)
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\actions\CInlineAction.php(49): WebsiteController->actionTest()
#1 E:\xampp_new\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): WebsiteController->runAction()
#3 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): WebsiteController->runActionWithFilters()
#4 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): WebsiteController->run()
#5 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/website/test
in E:\xampp_new\htdocs\YiiDemo\protected\controllers\WebsiteController.php (28)
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/24 17:02:19 [error] [exception.CException] exception 'CException' with message 'WebsiteController and its behaviors do not have a method or closure named "AbsoluteUrl".' in E:\xampp_new\htdocs\yii\framework\base\CComponent.php:265
Stack trace:
#0 E:\xampp_new\htdocs\YiiDemo\protected\controllers\WebsiteController.php(29): CComponent->__call('AbsoluteUrl', Array)
#1 E:\xampp_new\htdocs\YiiDemo\protected\controllers\WebsiteController.php(29): WebsiteController->AbsoluteUrl('website/page', Array)
#2 E:\xampp_new\htdocs\yii\framework\web\actions\CInlineAction.php(49): WebsiteController->actionTest()
#3 E:\xampp_new\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('test')
#7 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('website/test')
#8 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/YiiDemo/website/test
---
2013/02/25 10:27:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "test".' in E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('test')
#1 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/YiiDemo/test
---
2013/02/25 17:48:27 [error] [exception.CException] exception 'CException' with message 'TestController cannot find the requested view "test".' in E:\xampp_new\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('test', NULL, true)
#1 E:\xampp_new\htdocs\YiiDemo\protected\controllers\TestController.php(15): CController->render('test')
#2 E:\xampp_new\htdocs\yii\framework\web\actions\CInlineAction.php(49): TestController->actionIndex()
#3 E:\xampp_new\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('test')
#8 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/YiiDemo/test
---
2013/02/26 16:33:36 [error] [php] include(ModuleUrlManager.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (E:\xampp_new\htdocs\yii\framework\YiiBase.php:421)
Stack trace:
#0 unknown(0): spl_autoload_call()
#1 E:\xampp_new\htdocs\yii\framework\base\CComponent.php(556): call_user_func()
#2 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(196): CWebApplication->raiseEvent()
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(167): CWebApplication->onBeginRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/26 18:10:18 [error] [php] include(ModuleUrlManager.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (E:\xampp_new\htdocs\yii\framework\YiiBase.php:421)
Stack trace:
#0 unknown(0): spl_autoload_call()
#1 E:\xampp_new\htdocs\yii\framework\base\CComponent.php(556): call_user_func()
#2 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(196): CWebApplication->raiseEvent()
#3 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(167): CWebApplication->onBeginRequest()
#4 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/26 18:11:40 [error] [php] call_user_func() expects parameter 1 to be a valid callback, class 'ModuleUrlManager' not found (E:\xampp_new\htdocs\yii\framework\base\CComponent.php:556)
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(167): CWebApplication->onBeginRequest()
#1 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/26 18:12:43 [error] [php] call_user_func() expects parameter 1 to be a valid callback, class 'ModuleUrlManager' not found (E:\xampp_new\htdocs\yii\framework\base\CComponent.php:556)
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(167): CWebApplication->onBeginRequest()
#1 E:\xampp_new\htdocs\YiiDemo\index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/index.php/gii/
in E:\xampp_new\htdocs\YiiDemo\index.php (13)
2013/02/26 18:14:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection.connectionString cannot be empty.' in E:\xampp_new\htdocs\yii\framework\db\CDbConnection.php:369
Stack trace:
#0 E:\xampp_new\htdocs\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 E:\xampp_new\htdocs\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 E:\xampp_new\htdocs\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 E:\xampp_new\htdocs\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 E:\xampp_new\htdocs\yii\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 E:\xampp_new\htdocs\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 E:\xampp_new\htdocs\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 E:\xampp_new\htdocs\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 E:\xampp_new\htdocs\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 E:\xampp_new\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 E:\xampp_new\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 E:\xampp_new\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 E:\xampp_new\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 E:\xampp_new\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 E:\xampp_new\htdocs\YiiDemo\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/YiiDemo/gii/model
---
2013/03/01 08:43:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "users".' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/YiiDemo/users/
---
2013/03/01 08:43:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index".' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php:483
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(270): CController->missingAction('')
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('')
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user')
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/YiiDemo/user/
---
2013/03/01 08:44:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(438): CModule->getComponent('db')
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php(623): CApplication->getDb()
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/models/Post.php(21): CActiveRecord::model('Post')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php(13): Post::model()
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): PostController->actionIndex()
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#12 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('')
#14 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post')
#15 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#16 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/YiiDemo/post
---
2013/03/01 08:44:51 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii_external_actions.Post' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `Post`.
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/models/Post.php (21)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php (13)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/01 08:44:51 [error] [exception.CDbException] exception 'CDbException' with message 'The table "Post" for active record class "Post" cannot be found in the database.' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php:2310
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/models/Post.php(21): CActiveRecord::model('Post')
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php(13): Post::model()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): PostController->actionIndex()
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post')
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/YiiDemo/post
---
2013/03/01 08:45:32 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii_external_actions.Post' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `Post`.
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/models/Post.php (21)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php (13)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/01 08:45:32 [error] [exception.CDbException] exception 'CDbException' with message 'The table "Post" for active record class "Post" cannot be found in the database.' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php:2310
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/models/Post.php(21): CActiveRecord::model('Post')
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php(13): Post::model()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): PostController->actionIndex()
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post')
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/YiiDemo/post
---
2013/03/01 08:45:33 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii_external_actions.Post' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `Post`.
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/models/Post.php (21)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php (13)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/01 08:45:33 [error] [exception.CDbException] exception 'CDbException' with message 'The table "Post" for active record class "Post" cannot be found in the database.' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php:2310
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/models/Post.php(21): CActiveRecord::model('Post')
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php(13): Post::model()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): PostController->actionIndex()
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post')
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/YiiDemo/post
---
2013/03/01 08:45:34 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii_external_actions.Post' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `Post`.
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/models/Post.php (21)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php (13)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/01 08:45:34 [error] [exception.CDbException] exception 'CDbException' with message 'The table "Post" for active record class "Post" cannot be found in the database.' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php:2310
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/models/Post.php(21): CActiveRecord::model('Post')
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php(13): Post::model()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): PostController->actionIndex()
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post')
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/YiiDemo/post
---
2013/03/01 08:45:34 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii_external_actions.Post' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `Post`.
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/models/Post.php (21)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php (13)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/01 08:45:34 [error] [exception.CDbException] exception 'CDbException' with message 'The table "Post" for active record class "Post" cannot be found in the database.' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php:2310
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/ar/CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/models/Post.php(21): CActiveRecord::model('Post')
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php(13): Post::model()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): PostController->actionIndex()
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post')
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/YiiDemo/post
---
2013/03/01 08:50:54 [error] [exception.CException] exception 'CException' with message 'PostController cannot find the requested view "index".' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php:878
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php(16): CController->render('index', Array)
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): PostController->actionIndex()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('')
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/YiiDemo/post
---
2013/03/01 08:54:20 [error] [php] Undefined variable: post (/Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/views/post/index.php:10)
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(869): PostController->renderFile()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(782): PostController->renderPartial()
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php(16): PostController->render()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): PostController->actionIndex()
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): PostController->runAction()
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): PostController->runActionWithFilters()
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): PostController->run()
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/post
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/views/post/index.php (10)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php (16)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/01 08:54:33 [error] [php] Use of undefined constant posts - assumed 'posts' (/Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/views/post/index.php:10)
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(869): PostController->renderFile()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(782): PostController->renderPartial()
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php(16): PostController->render()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): PostController->actionIndex()
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): PostController->runAction()
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): PostController->runActionWithFilters()
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): PostController->run()
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/post
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/views/post/index.php (10)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/PostController.php (16)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/01 09:02:01 [error] [exception.CHttpException.404] exception 'CHttpException' in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php:15
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CAction.php(76): DeleteAction->run()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CAction->runWithParams(Array)
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(DeleteAction))
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(DeleteAction), Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('delete')
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post/delete')
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/YiiDemo/post/delete
---
2013/03/01 09:11:01 [error] [exception.CHttpException.404] exception 'CHttpException' in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php:15
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CAction.php(76): DeleteAction->run()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CAction->runWithParams(Array)
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(DeleteAction))
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(DeleteAction), Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('delete')
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post/delete/1')
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/YiiDemo/post/delete/1
---
2013/03/01 09:12:56 [error] [php] Undefined index: delete (/Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php:18)
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): PostController->runAction()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): PostController->runActionWithFilters()
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): PostController->run()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/post/delete/id/1
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/01 09:14:43 [error] [exception.CException] exception 'CException' with message 'DeleteAction and its behaviors do not have a method or closure named "redirect".' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CComponent.php:265
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php(24): CComponent->__call('redirect', Array)
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php(24): DeleteAction->redirect('index')
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CAction.php(76): DeleteAction->run()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CAction->runWithParams(Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(DeleteAction))
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(DeleteAction), Array)
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('delete')
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post/delete/id/...')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/YiiDemo/post/delete/id/1
---
2013/03/01 09:14:46 [error] [exception.CHttpException.404] exception 'CHttpException' in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php:21
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CAction.php(76): DeleteAction->run()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CAction->runWithParams(Array)
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(DeleteAction))
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(DeleteAction), Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('delete')
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post/delete/id/...')
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/YiiDemo/post/delete/id/1
---
2013/03/01 09:15:08 [error] [exception.CException] exception 'CException' with message 'DeleteAction and its behaviors do not have a method or closure named "redirect".' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CComponent.php:265
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php(24): CComponent->__call('redirect', Array)
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php(24): DeleteAction->redirect('index')
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CAction.php(76): DeleteAction->run()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CAction->runWithParams(Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(DeleteAction))
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(DeleteAction), Array)
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('delete')
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post/delete/id/...')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/YiiDemo/post/delete/id/2
---
2013/03/01 09:18:09 [error] [exception.CHttpException.404] exception 'CHttpException' in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php:21
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CAction.php(76): DeleteAction->run()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CAction->runWithParams(Array)
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(DeleteAction))
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(DeleteAction), Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('delete')
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post/delete/id/...')
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/YiiDemo/post/delete/id/2
---
2013/03/01 09:18:26 [error] [exception.CException] exception 'CException' with message 'DeleteAction and its behaviors do not have a method or closure named "redirect".' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CComponent.php:265
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php(24): CComponent->__call('redirect', Array)
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php(24): DeleteAction->redirect('index')
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CAction.php(76): DeleteAction->run()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CAction->runWithParams(Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(DeleteAction))
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(DeleteAction), Array)
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('delete')
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post/delete/id/...')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/YiiDemo/post/delete/id/3
---
2013/03/01 09:23:15 [error] [exception.CHttpException.404] exception 'CHttpException' in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php:21
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CAction.php(76): DeleteAction->run()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CAction->runWithParams(Array)
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(DeleteAction))
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(DeleteAction), Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('delete')
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post/delete/id/...')
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/YiiDemo/post/delete/id/3
---
2013/03/01 09:23:27 [error] [exception.CHttpException.404] exception 'CHttpException' in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php:21
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CAction.php(76): DeleteAction->run()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CAction->runWithParams(Array)
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(DeleteAction))
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(DeleteAction), Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('delete')
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post/delete/id/...')
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/YiiDemo/post/delete/id/index
---
2013/03/01 09:25:49 [error] [exception.CHttpException.404] exception 'CHttpException' in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/components/DeleteAction.php:21
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CAction.php(76): DeleteAction->run()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CAction->runWithParams(Array)
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(DeleteAction))
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(DeleteAction), Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('delete')
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('post/delete/id/...')
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/YiiDemo/post/delete/id/post/index
---
2013/03/04 11:05:01 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('')
#13 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/YiiDemo/gii/model
HTTP_REFERER=http://localhost/YiiDemo/gii/default/index
---
2013/03/04 16:57:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "taks".' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('taks')
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/YiiDemo/taks
---
2013/03/04 16:57:26 [error] [exception.CException] exception 'CException' with message 'Property "Task.tittle" is not defined.' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CComponent.php:130
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/helpers/CHtml.php(2255): CComponent->__get('tittle')
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/helpers/CHtml.php(1980): CHtml::resolveValue(Object(Task), 'tittle')
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/helpers/CHtml.php(1257): CHtml::activeInputField('text', Object(Task), 'tittle', Array)
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/views/task/_task.php(4): CHtml::activeTextField(Object(Task), '[0]tittle')
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CBaseController.php(126): require('/Users/ducnguye...')
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/Users/ducnguye...', Array, true)
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(869): CBaseController->renderFile('/Users/ducnguye...', Array, true)
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/views/task/index.php(8): CController->renderPartial('_task', Array)
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CBaseController.php(126): require('/Users/ducnguye...')
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/Users/ducnguye...', Array, true)
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(869): CBaseController->renderFile('/Users/ducnguye...', Array, true)
#11 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(782): CController->renderPartial('index', Array, true)
#12 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/TaskController.php(37): CController->render('index', Array)
#13 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): TaskController->actionIndex()
#14 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#15 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#16 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(291): CFilterChain->run()
#17 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('')
#19 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('task')
#20 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#21 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/YiiDemo/task
---
2013/03/04 17:07:16 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php:336
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(1130): CFilterChain->run()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAjaxOnly(Object(CFilterChain))
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('field')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('task/field')
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/YiiDemo/task/field?_=1362391636767
HTTP_REFERER=http://localhost/YiiDemo/task
---
2013/03/04 17:07:31 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php:336
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(1130): CFilterChain->run()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAjaxOnly(Object(CFilterChain))
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('field')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('task/field')
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/YiiDemo/task/field?_=1362391651786
HTTP_REFERER=http://localhost/YiiDemo/task
---
2013/03/04 19:57:39 [error] [php] Undefined variable: uploaded (/Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/views/upload/index.php:1)
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(869): UploadController->renderFile()
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(782): UploadController->renderPartial()
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php(29): UploadController->render()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): UploadController->actionIndex()
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): UploadController->runAction()
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): UploadController->runActionWithFilters()
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): UploadController->run()
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CWebApplication->run()
REQUEST_URI=/YiiDemo/upload
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/views/upload/index.php (1)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (29)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 19:58:07 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 19:59:03 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 19:59:35 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 20:14:33 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 20:18:20 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 20:19:47 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 20:20:10 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 20:20:32 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 20:20:39 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 20:21:03 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 20:21:59 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 20:23:02 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (19)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 20:26:49 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (19)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
2013/03/04 20:26:49 [error] [exception.CException] exception 'CException' with message 'CUploadedFile and its behaviors do not have a method or closure named "name".' in /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CComponent.php:265
Stack trace:
#0 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php(26): CComponent->__call('name', Array)
#1 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php(26): CUploadedFile->name()
#2 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/actions/CInlineAction.php(49): UploadController->actionIndex()
#3 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(282): CController->run('')
#7 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('upload')
#8 /Users/ducnguyenhuy/MyProjects/nerdwallet/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/YiiDemo/upload
HTTP_REFERER=http://localhost/YiiDemo/upload
---
2013/03/04 20:27:54 [warning] [application] Failed to set unsafe attribute "file" of "Upload".
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/protected/controllers/UploadController.php (18)
in /Users/ducnguyenhuy/MyProjects/nerdwallet/YiiDemo/index.php (13)
